config/html-elements-plugin/index.js

```diff
@@ -0,0 +1,104 @@
+
+function HtmlElementsPlugin(locations) {
+  this.locations = locations;
+}
+
+HtmlElementsPlugin.prototype.apply = function(compiler) {
```

Comment:
after this is merged can we you create a module `html-head-plugin` and import it in this project?